scripts/update-local-config.js

```diff
@@ -23,9 +25,15 @@ function showUsage({ alreadyLoggedIn = true } = {}) {
   /* eslint-enable no-console */
 }
 
-function updatedLocalConfig(vcapJson) {
+function getUpsCreds(vcapJson) {
+  return vcapJson.VCAP_SERVICES['user-provided']
+    .find(el => el.instance_name === credsUpsName)
+    .credentials;
+}
+
+function updateLocalConfig(vcapJson) {
```

Comment:
Is it safe to assume that the credentials object will also always be at this index?

Reply:
Yeah, all of the entries in the `user-provided` array will have a `credentials` property.

I didn't put much error handling in this code at all, because I figured if it broke, it wasn't a big deal since it just impacts a local dev environment, and probably indicates something else weird going on.